aws/cloudformation/bootstrap_chef_stack.sh.erb

```diff
@@ -4,10 +4,25 @@ unattended-upgrade
 apt-get -y install python3-pip curl
 test "$(pip3 show awscli)" || pip3 install awscli
 
+# Environment variables set via CloudFormation string substitution: !Sub encloses the output of this ERB template
+# https://github.com/code-dot-org/code-dot-org/blob/1f98f3403ff1f1f1c64b94eeb4c70c9584ae62aa/aws/cloudformation/cloud_formation_stack.yml.erb#L535-L543
 STACK=${AWS::StackName}
-CHEF_VERSION=<%=CHEF_VERSION%>
 REGION=${AWS::Region}
 BRANCH=${Branch}
+BASE_DOMAIN_NAME=${BaseDomainName}
+DASHBOARD_SUBDOMAIN_NAME=${DashboardSubdomainName}
+PEGASUS_SUBDOMAIN_NAME=${PegasusSubdomainName}
+# Bash and CloudFormation use the same string interpolation syntax. Escape with "!" to ensure CloudFormation
+# does not modify the bash syntax.
+DASHBOARD_DOMAIN_NAME="${!DASHBOARD_SUBDOMAIN_NAME}.${!BASE_DOMAIN_NAME}"
+# In production the fully qualified Pegasus domain name is just "code.org"; detect if the Pegasus sub domain is empty.
+[ -z "$PEGASUS_SUBDOMAIN_NAME" ] && PEGASUS_DOMAIN_NAME=$BASE_DOMAIN_NAME || PEGASUS_DOMAIN_NAME="${!PEGASUS_SUBDOMAIN_NAME}.${!BASE_DOMAIN_NAME}"
+# TODO: Remove these debug statements.
+echo $PEGASUS_DOMAIN_NAME
+echo $DASHBOARD_DOMAIN_NAME
```

Comment:
nit: stray debug statements

Reply:
Yup. Will remove after the current adhoc finishes provisioning. This logic doesn't get tested in continuous integration builds.